src/types/platforms-types/heds-collab.ts

```diff
@@ -0,0 +1,142 @@
+import _ from 'lodash';
+import slugify from 'slugify';
+
+
+import { extractHashFromURL } from '../../clients/ipfs';
+import { formatAddress } from '../address';
+import { ArtistProfile } from '../artist';
+import { NFT, getTrait, NftFactory } from '../nft';
+import { MapTrack } from '../processor';
+import { ProcessedTrack } from '../track';
+
+function getTrackNameFromArtist(artist: string): string {
+  const trackByArtist: any = {
+    'LOPHIILE FT. TYLER JAY': 'see thru',
+    'CARRTOONS': 'Cahuenga',
+    'ROMDERFUL': 'DO YOU WANT TO TAKE A FLIGHT?',
+    'AUTUMN KEYS FT AYOTEMI': 'Same Old',
+    'OSHI': 'family photos',
+    'DECAP': 'Gates',
+    'CHROMONICCI': `Cosmo’s Adventure`,
+    'XANDER': 'Somebody Else',
+    'CAPSHUN & CORDEROYBOI': 'FINE',
+    'PAUL MOND': 'is you real'
+  }
+
+  const track = trackByArtist[artist.toUpperCase()];
+
+  if (!track){
+    throw `Can't map artist to track name`;
+  }
+
+  return track
+
+}
+
+function getAvatarFromArtist(artist: string): string {
+  const avatarByArtist: any = {
+    'LOPHIILE FT. TYLER JAY': 'https://firebasestorage.googleapis.com/v0/b/heds-34ac0.appspot.com/o/tapes%2FcollabTAPES%2FGood%20Society%2FImages%2Fartists%2FLophiile.jpg?alt=media&token=538507ad-08d7-499d-a7af-f689377f3349',
```

Comment:
these urls look risky, especially with there being an access token in them. lets upload to ipfs and use those